combat-trainer.lic

```diff
@@ -4048,18 +4065,45 @@ class GameState
     end
   end
 
+  # Get the skill of the weapon currently being trained (e.g. Small Edged).
   def weapon_skill
     @current_weapon_skill
   end
 
+  # Does the current weapon train a two-handed skill (i.e. 2HE/2HB)?
   def twohanded_weapon_skill?
     $twohanded_skills.include?(weapon_skill)
   end
 
+  # Does the current weapon train a non-brawling melee skill (i.e. edged, blunt, staff, polearm)?
   def melee_weapon_skill?
     $melee_skills.include?(weapon_skill)
   end
 
+  # Does the current weapon train a thrown skill (i.e. light/heavy thrown)?
+  def thrown_skill?
+    $thrown_skills.include?(weapon_skill)
+  end
```

Comment:
Lines 4083-4105 are existing methods that have been _moved_ to here so all these `xyz_weapon_skill?` methods are co-located.